src/allotropy/parsers/agilent_gen5_image/agilent_gen5_image_structure.py

```diff
@@ -250,11 +250,32 @@ def _get_read_type(cls, procedure_details: str) -> ReadType:
 
 
 def create_results(
-    result_lines: list[str],
+    result_lines: list[str] | None,
     header_data: HeaderData,
     read_data: ReadData,
     sample_identifiers: dict[str, str],
 ) -> list[MeasurementGroup]:
+    if not result_lines:
+        return [
+            MeasurementGroup(
+                measurement_time=header_data.datetime,
+                plate_well_count=0,
```

Comment:
I think we can still read plate well count. I see 96 in "Plate Type" in the header

Perhaps we need to adjust our plate_well_count strategy for this adapter to read from there.

Reply:
@nathan-stender, believe that we decided not to use "Plate Type" as the plate well count reference since the instrument file can have other vendor specific names for plates that don't include the number of wells. 

Logic could be for these examples is to look for well number if present in "Plate Type" field, record "N/A" if not found. Think best approach is still to pull from the plate well matrix, when available.

Reply:
I see issue with "N/A" now being that the field requires numeric value. Have to think about that implementation

Reply:
@james-leinas I thought what we landed on was a fallback strategy:

1. Read from plate data matrix (current approach) if at all possible
2. Try to read from Plate Type if there is a number string in the plate type. Given that this is customer set and we don't want to record any-old thing (like if they name the plate `Plate type 2` or something we don't want to pull 2) perhaps this can just search for some expected numbers, like 96 and 384?
3. Record 0, as (@joshua-benchling this is a numeric-only field in ASM) - honestly at this point I doubt it's that big of a deal, for this use case customer probably doesn't care about it?